system/HTTP/CURLRequest.php

```diff
@@ -674,8 +674,16 @@ protected function setCURLOptions(array $curl_options = [], array $config = [])
 		// Debug
 		if (isset($config['debug']))
 		{
-			$curl_options[CURLOPT_VERBOSE] = $config['debug'] === true ? 1 : 0;
-			$curl_options[CURLOPT_STDERR]  = $config['debug'] === true ? fopen('php://output', 'w+') : $config['debug'];
+			if(is_bool($config['debug']) && $config['debug'] === true)
+			{
+				$curl_options[CURLOPT_VERBOSE] = 1;
+				$curl_options[CURLOPT_STDERR] = fopen('php://output', 'w+');
+			}
+			else if(is_resource($config['debug']))
```

Comment:
I'm confused. When is `$config['debug']` going to be a resource? According to the docs it should only ever be a boolean. If true, it turns on verbose and sets the error output, otherwise it leaves verbose to whatever it was otherwise set at.